utils/certs/gen_certificates.sh

```diff
@@ -206,6 +221,8 @@ function generate_test_cert () {
         -cert "${CERTS}/daosCA.crt" -policy signing_policy \
         -extensions signing_test -out "${CERTS}/test.crt" \
         -outdir "${CERTS}" -in "${CA_HOME}/test.csr" -batch
+    # TODO [[ $EUID -eq 0 ]] && chown ?.? "${CERTS}/test.crt"
+    chmod 0644 "${CERTS}/test.crt"
```

Comment:
@brianjmurrell @dinghwah Could you please advise what user we should use for the test.crt?

Reply:
I don't know what it's used for.  Perhaps @dinghwah could explain.

Reply:
I do think we should remove test.crt and related lines now.
test.crt was used in the early phase for testing propose.

Reply:
OK I will do that now and push again, thanks!